lib/template.html

```diff
@@ -51,7 +51,11 @@
 
 {% if site.twitter %}
   {% if seo_tag.image %}
-    <meta name="twitter:card" content="summary_large_image" />
+    {% if page.twitter_image_small or post.twitter_image_small %}
+      <meta name="twitter:card" content="summary" />
+    {% else %}
+      <meta name="twitter:card" content="summary_large_image" />
+    {% endif %}
```

Comment:
App and player cards require other meta data info rather than just the image, do there is quite a bit more effort required.
For a simple image based card which is what this section has, there is just small or big image.